src/cache.ts

```diff
@@ -42,23 +43,21 @@ const createQueryFilterFromSpec = <Endpoints extends RoughEndpoints>(
     }),
 });
 
+export const infinite = 'infinite' as const;
```

Comment:
By exporting these here, it's going to be added to the `index` export of the package. Since that makes it pretty public, could we name it using a more standard constant syntax? Maybe like `INFINITE_QUERY_KEY`?